ra/ra.go

```diff
@@ -880,7 +877,8 @@ func mergeUpdate(r *core.Registration, input core.Registration) bool {
 		changed = true
 	}
 
-	if features.Enabled(features.AllowKeyRollover) && input.Key != emptyKey && input.Key != r.Key {
+	if features.Enabled(features.AllowKeyRollover) &&
```

Comment:
Instead, can we change the field in `Registration` to be a `*jose.JsonWebKey` and check directly if it is `nil`?